vsintegration/ProjectTemplates/NetCore259Project/Template/PortableLibrary.fsproj

```diff
@@ -53,7 +53,10 @@
   <PropertyGroup>
     <MinimumVisualStudioVersion Condition="'$(MinimumVisualStudioVersion)' == ''">12</MinimumVisualStudioVersion>
   </PropertyGroup>
-  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\FSharp\Microsoft.Portable.FSharp.Targets" />
+  <PropertyGroup>
+    <PortableFSharpTargetsPath Condition=" '$(PortableFSharpTargetsPath)' == '' ">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\FSharp\Microsoft.Portable.FSharp.Targets</PortableFSharpTargetsPath>
+  </PropertyGroup>
+  <Import Project="$(PortableFSharpTargetsPath)" />
```

Comment:
As long as there is any way to override these targets files, I do not really care about if there is one variable or two. But ``FSharpTargetsPath`` has historically pointed directly to the file, and a few people have already manually changed their project files and hardcoded this. So I think this would cause a bit of confusion, and the whole topic is already complex enough.

So if we want to use a single variable (which I actually also like), then I think we should use a different name, e.g. ``FSharpTargetsDir``, to make clear it points to the directory. @KevinRansom  What do you think?